src/main/java/amazin/model/Book.java

```diff
@@ -12,7 +12,7 @@
 
 @Entity
 @Indexed
-@Table(name="book")
+@Table(name="book_table")
```

Comment:
renaming the table is not necessary. table names don't typically have a _table prefix.

Reply:
I was following convention here since I think I had to rename the user table to user_table since user seems to be a reserved keyword in postgreSQL. I figured instead of only having the user table with the _table prefix, I would make all of them have the _table prefix.